src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {useState, useContext} from "react";
+import { UserContext } from '../../context/userContext'
+
+import "./Profile.css";
+
+function Profile() {
+    const user = useContext(UserContext);
+    return (
+        <main>
+            <h1>Profile</h1>
+            <div>{user && <p>Username: {user.username}</p>}</div>
+            <div>{user && <p>Bio: {user.bio}</p>}</div>
+            <div>{user && <p>Genres: {user.genres}</p>}</div>
```

Comment:
I would display the posts that the user makes here as well to display more content and not have a big white screen. Or make the font bigger. Lots of space to play around with styles too.

Just want to avoid 
![image](https://github.com/user-attachments/assets/27d51489-0c46-4c1b-b648-31691f2fe4e3)